addons/account/wizard/account_payment_register.py

```diff
@@ -902,7 +902,7 @@ def _post_payments(self, to_process, edit_mode=False):
         payments = self.env['account.payment']
         for vals in to_process:
             payments |= vals['payment']
-        payments.action_post()
+        payments.with_context(skip_invoice_send=True).action_post()
```

Comment:
I don't like adding context keys, but well, I don't see much choice here.
Please mention `sale` somehow, either in the context key name, or add a comment.
Like, `skip_sale_auto_invoice` or something